test/unit/vdom/skip.js

```diff
@@ -10,6 +10,7 @@ describe('vdom/skip', () => {
       },
       render() {
         vdom.element('div', { skip: true }, () => {
+          vdom.text('text');
```

Comment:
Yeah, but more importantly immediate children in general. That's what was broken. I should probably be checking `innerHTML` rather that `textContent` below to test both.